ui/components/policies/policy-view.jsx

```diff
@@ -16,18 +16,18 @@ export default function Policy({ policy, topicsIds }) {
     },
   };
   return (
-    <li key={policy.id} className="py1">
+    <li key={policy.id} className="my2">
       <section className="border rounded gray-border p2">
         <h2 className="mt0 mb3">{policy.title_with_number}</h2>
         <div className="clearfix">
           <span className="requirements-links col col-6">
-            <span className="circle-bg border gray-border p1">
+            <div className="circle-bg border gray-border center p1">
               <Link to={relevantReqs}>
                 {policy.relevant_reqs}
               </Link>
-            </span> of <Link to={allReqs}>
-              {policy.total_reqs} requirements
-            </Link> match your search
+            </div> of&nbsp;
+            {policy.total_reqs} requirements
+             match your search
```

Comment:
One of the desires was to clarify action items.
https://github.com/18F/omb-eregs/issues/298